ocs_ci/deployment/aws.py

```diff
@@ -569,9 +574,12 @@ def run_ansible_playbook(self):
         pem_dst_path = "/openshift-dev.pem"
         pod.upload(rhel_pod_obj.name, pem_src_path, pem_dst_path)
         repo_dst_path = "/etc/yum.repos.d/"
-        repo_file = os.path.basename(constants.OCP4_2_REPO)
+        repo = os.path.join(
+            constants.REPO_DIR, f"ocp_{get_ocp_version('_')}.repo"
+        )
+        repo_file = os.path.basename(repo)
```

Comment:
Don't we want to check if `repo` file exists? Something like:
```python
assert os.path.exists(repo), f"Required repo file {repo} doesn't exist!"
```

Reply:
done